packages/EasyCodingStandard/config/clean-code.yml

```diff
@@ -2,6 +2,9 @@ services:
     SlevomatCodingStandard\Sniffs\Variables\UnusedVariableSniff: ~
     SlevomatCodingStandard\Sniffs\Variables\UselessVariableSniff: ~
 
+    # detect unused variables passed to closures via use
```

Comment:
Could you add super simple code here instead? I didn't get it from the description nor the class name and had to try it.

Something like:

```
function ($var) use ($unused) { return $var }    →   function ($var) { return $var } 
```